frontend/src/components/doctors/doctors/DoctorDistance.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+    DoctorLocation,
+    getDoctorLocationDistance,
+} from "../../../types/doctors/doctor";
+import { Location } from "../../../utils/googleMaps/useGoogleMaps";
+import { DistanceUnit } from "../../utils/DistanceUnit";
+import Icon from "../../utils/Icon";
+
+interface DoctorDistanceProps {
+    location: DoctorLocation | null | undefined;
```

Comment:
Please call It doctor location.
Location is too general and we have another location here.